src/LoginForm.jsx

```diff
@@ -107,7 +107,7 @@ class LoginForm extends React.Component {
   };
 
   componentWillUnmount = () => {
-    this._cleanup();
+    this._cleanup().then(() => console.log('cleanup complete'))
```

Comment:
`_cleanup()` is declared as `async` method, so just calling it won't do anything. Also there's multiple `await` inside of it, so I'm not sure how to `await` all of them here.